_posts/en/pages/2016-01-18-segwit-wallet-dev.md

```diff
@@ -0,0 +1,124 @@
+---
+version: 1
+title: Segregated Witness Wallet Development Guide
+name: segwit-wallet-dev
+type: page
+permalink: /en/segwit_wallet_dev
+---
+
+### Witness program
+
+There are 2 ways to use segregated witness:
+
+#### Native witness program
+
+Native witness program is defined as a <code>scriptPubKey</code> with a single byte push (<code>OP_0</code>, <code>OP_1</code>, ...,  <code>OP_16</code>) followed by a push of 2 to 32 bytes.
+
+#### Witness program nested in P2SH
```

Comment:
Would be good to provide a link to P2SH explanation, https://bitcoin.org/en/developer-guide#p2sh-scripts